.azure-pipelines/gpu-tests.yml

```diff
@@ -59,6 +59,11 @@ jobs:
         python -m coverage run --source flash -m pytest flash tests/examples/test_scripts.py -v --junitxml=$(Build.StagingDirectory)/test-results.xml --durations=30
       displayName: 'Testing'
 
+    - bash: |
+        pip install git+https://github.com/tchaton/learn2learn@flash
+        bash tests/special_tests.sh
+      displayName: 'Testing: special'
```

Comment:
Currently our GPU tests only run nightly. Should these special tests run every PR instead? We could temp enable them here just to at least check that everything passes

Reply:
We can enable this later. I need to check with Jirka if it is worth to enable it on every PR.